backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/objectal/OALIOSAudio.java

```diff
@@ -49,13 +53,20 @@ public void activate () {
 	}
 
 	@Override
-	public void deactivate () {
+	public void willEnterForeground () {
 		// workaround for ObjectAL crash problem
 		// see: https://groups.google.com/forum/?fromgroups=#!topic/objectal-for-iphone/ubRWltp_i1Q
-		OALAudioSession audioSession = OALAudioSession.sharedInstance();
-		if (audioSession != null) {
-			audioSession.forceEndInterruption();
-		}
+		forceEndInterruption();
+	}
+
+	@Override
+	public void willResignActive () {
+
+	}
+
+	@Override
+	public void willTerminate () {
+
 	}
```

Comment:
Just a method reordering suggestion, can you move the new `IOSAudio` methods after the `Audio` methods and the `forceEndInterruption()` private method at the bottom so that the constructor is on top.

Reply:
Yes, that makes sense.